default/python/turn_events/turn_events.py

```diff
@@ -6,6 +6,66 @@
 from universe_tables import MONSTER_FREQUENCY
 
 
+
+# tuples of galaxy setup options to pick from randomly
+low_to_high =  (fo.galaxySetupOption.low, fo.galaxySetupOption.medium, fo.galaxySetupOption.high)
+none_to_high = (fo.galaxySetupOption.low, fo.galaxySetupOption.medium, fo.galaxySetupOption.high, fo.galaxySetupOption.none)
+
+class PyGalaxySetupData:
```

Comment:
I needed the same functionality somewhere else, but don't know where/how to move the code to be accessible from both places.